docs/tutorial.rst

```diff
@@ -102,11 +104,415 @@ mlf-core is now configured and ready to be used!
+    RUN conda env create -f environment.yml && conda clean -a
+
+    # Activate the environment
+    RUN echo "source activate exploding_springfield" >> ~/.bashrc
+    ENV PATH /home/user/miniconda/envs/exploding_springfield/bin:$PATH
+
+    # Dump the details of the installed packages to a file for posterity
+    RUN conda env export --name exploding_springfield > exploding_springfield_environment.yml
+
+The Docker container simply uses the environment.yml file to create a Conda environment and activates it.
+You can find the base container definitions in the `mlf-core containers repository <https://github.com/mlf-core/containers>`_.
+
+environment.yml
+~~~~~~~~~~~~~~~~
+
+The ``environment.yml`` file is used for both, running the mlf-core project with Conda, and for creating the Conda environment inside the Docker container.
+Therefore you only need to specify your dependencies once in this file.
+Try to always define all dependencies from Conda channels if possible and only add PyPI dependencies if a Conda version is not available.
+
+.. code-block::
+
+    name: project_name
+    channels:
+    - defaults
+    - conda-forge
+    - pytorch
+    dependencies:
+    - defaults::cudatoolkit=11.0.221
+    - defaults::python=3.8.2
+    - conda-forge::tensorboardx=2.1
+    - conda-forge::mlflow=1.13.1
+    - conda-forge::rich=9.10.0
+    - pytorch::pytorch=1.7.1
+    - pytorch::torchvision=0.8.2
+    - pytorch-lightning==1.1.8
+    - pip
+    - pip:
+        - pycuda==2019.1.2  # not on Conda
+        - cloudpickle==1.6.0
+        - boto3==1.17.7
+        - system-intelligence==2.0.2
+
+If you have dependencies that are not available on Conda nor PyPI you can adapt the Docker container.
+
+Post project creation TODOs
+---------------------------------------
+
+mlf-core tries to automate as much as possible, but some minor actions need to be done manually.
+
+Public Docker container on Github Packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+mlf-core by default pushes the Docker container using the ``publish_docker.yml`` Github Actions workflow to `Github Packages <https://github.com/features/packages>`_.
+If you want to push your Docker container to a different registry you need to adapt the workflow and potentially update the username and add a Github secret for your password.
+By default, containers pushed to Github are private. As a result you would need to log in to pull the container.
+Hence, you have to make your Docker container public by navigating to the used Github account, selecting ``Packages`` and then your package.
+On the right you will find a button ``package settings``. Scroll down on the package settings page and at the bottom you will find a button ``make public``.
+Click it, authenticate and your Github container is now public!
+Be aware of the fact that building the Docker container usually takes 15-20 minutes and therefore your Docker container will not immidiately show up in the Packages tab.
```

Comment:
Maybe add an image like u did it above with the secret? Would improve "readability" a lot.